bicep/workload-stamp.bicep

```
@@ -0,0 +1,639 @@
+    app: 'fabrikam-ingestion and fabrikam-workflow'
+    'app-producer': 'fabrikam-ingestion'
+    'app-consumer': 'fabrikam-workflow'
+  }
+}
+
+resource ingestionSBNamespace_ingestionSB 'Microsoft.ServiceBus/namespaces/queues@2022-10-01-preview' = {
+  parent: ingestionSBNamespace
+  name: ingestionSBName
+  properties: {
+    lockDuration: 'PT5M'
+    maxSizeInMegabytes: 1024
+  }
+}
+
+resource ingestionSBNamespace_ingestionServiceAccessKey 'Microsoft.ServiceBus/namespaces/AuthorizationRules@2022-10-01-preview' = {
+  parent: ingestionSBNamespace
+  name: ingestionServiceAccessKeyName
+  properties: {
+    rights: [
+      'Send'
+    ]
+  }
+}
+
+resource ingestionSBNamespace_workflowServiceAccessKey 'Microsoft.ServiceBus/namespaces/AuthorizationRules@2022-10-01-preview' = {
+  parent: ingestionSBNamespace
+  name: workflowServiceAccessKeyName
+  properties: {
+    rights: [
+      'Listen'
+    ]
+  }
+}
+
+resource deliveryKeyVault 'Microsoft.KeyVault/vaults@2022-07-01' = {
+  name: deliveryKeyVaultName
+  location: location
+  tags: {
+    displayName: 'Delivery Key Vault'
+    app: 'fabrikam-delivery'
+  }
+  properties: {
+    sku: {
+      family: 'A'
+      name: 'standard'
+    }
+    tenantId: subscription().tenantId
+    networkAcls: {
+      bypass: 'AzureServices'
+      defaultAction: 'Allow'
+      virtualNetworkRules: []
+    }
+    accessPolicies: [
+      {
+        tenantId: subscription().tenantId
+        objectId: deliveryPrincipalId
+        permissions: {
+          secrets: [
+            'get'
+            'list'
+          ]
+        }
+      }
+    ]
+  }
+  dependsOn: []
+}
+
+resource deliveryKeyVaultName_CosmosDB_Endpoint 'Microsoft.KeyVault/vaults/secrets@2022-07-01' = {
+  parent: deliveryKeyVault
+  name: 'CosmosDB-Endpoint'
+  properties: {
+    value: deliveryCosmosDb.properties.documentEndpoint
+  }
+}
+
+resource deliveryKeyVaultName_CosmosDB_Key 'Microsoft.KeyVault/vaults/secrets@2022-07-01' = {
+  parent: deliveryKeyVault
+  name: 'CosmosDB-Key'
+  properties: {
+    value: listKeys(deliveryCosmosDb.id, '2016-03-31').primaryMasterKey
+  }
+}
+
+resource deliveryKeyVaultName_Redis_Endpoint 'Microsoft.KeyVault/vaults/secrets@2022-07-01' = {
+  parent: deliveryKeyVault
+  name: 'Redis-Endpoint'
+  properties: {
+    value: deliveryRedis.properties.hostName
+  }
+}
+
+resource deliveryKeyVaultName_Redis_AccessKey 'Microsoft.KeyVault/vaults/secrets@2022-07-01' = {
+  parent: deliveryKeyVault
+  name: 'Redis-AccessKey'
+  properties: {
+    value: listKeys(deliveryRedis.id, '2016-04-01').primaryKey
+  }
+}
+
+resource deliveryKeyVaultName_ApplicationInsights_InstrumentationKey 'Microsoft.KeyVault/vaults/secrets@2022-07-01' = {
+  parent: deliveryKeyVault
+  name: 'ApplicationInsights--InstrumentationKey'
+  properties: {
+    value: reference(appInsights.id, '2015-05-01').InstrumentationKey
+  }
+}
+
+resource packageKeyVault 'Microsoft.KeyVault/vaults@2022-07-01' = {
```

The convention changed: 

```suggestion
resource keyVaultPackage 'Microsoft.KeyVault/vaults@2022-07-01' = {
```